app/services/make_offer.rb

```diff
@@ -38,6 +38,7 @@ def save!
         ).call!
 
         SendNewOfferEmailToCandidate.new(application_choice:).call
+        NotifyOfUnaccreditedProviderRecruitment.new(application_choice: application_choice).check_for_unaccredited_provider
```

Comment:
`NotifyOfRecruitmentByClosedProviders`?

Reply:
Worth sticking to the calling convention on services which is `X.new(configuration_etc).call`

Reply:
🧹